src/app/accounts/login.component.ts

```diff
@@ -189,4 +210,56 @@ export class LoginComponent extends BaseLoginComponent implements OnDestroy {
         });
     }
 
+    /**
+     * This is an override of base login component
+     * It is needed to handle redirect uri if provided
+     */
+    async submit() {
+        if (this.email == null || this.email === '') {
+            this.platformUtilsService.showToast('error', this.i18nService.t('errorOccurred'),
+                this.i18nService.t('emailRequired'));
+            return;
+        }
+        if (this.email.indexOf('@') === -1) {
+            this.platformUtilsService.showToast('error', this.i18nService.t('errorOccurred'),
+                this.i18nService.t('invalidEmail'));
+            return;
+        }
+        if (this.masterPassword == null || this.masterPassword === '') {
```

Comment:
nit: `!this.masterPassword` pourrait suffire non ?

Reply:
Possible oui, mais là c'est du code dupliqué de la fonction parente Bitwarden, donc je ne préfère y toucher que si nécessaire.